test/e2e/redundancy/gateway_failover.go

```diff
@@ -32,8 +32,54 @@ var _ = Describe("[redundancy] Gateway fail-over tests", func() {
 			testGatewayFailOverScenario(f)
 		})
 	})
+
+	When("a Route-agent pod is restarted on the Gateway node", func() {
+		It("should start a new submariner engine pod, and be able to connect from another cluster", func() {
+			testRouteAgentFailOverScenario(f)
+		})
+	})
+
 })
 
+// TODO : WIP flow: get route agent, delete , check connectivity
+func testRouteAgentFailOverScenario(f *subFramework.Framework) {
+	clusterAName := framework.TestContext.ClusterIDs[framework.ClusterA]
+	clusterBName := framework.TestContext.ClusterIDs[framework.ClusterB]
+
+	gatewayNodes := f.FindNodesByGatewayLabel(framework.ClusterA, true)
+	Expect(gatewayNodes).To(HaveLen(1), fmt.Sprintf("Expected only one gateway node on %q", clusterAName))
+	By(fmt.Sprintf("Found gateway on node %q on %q", gatewayNodes[0].Name, clusterAName))
+
+	routeAgentPod := f.AwaitRouteAgentPod(framework.ClusterA)
+	By(fmt.Sprintf("Found submariner route angine pod %q on %q", routeAgentPod.Name, clusterAName))
+
+	By(fmt.Sprintf("Deleting submariner route agent pod %q", routeAgentPod.Name))
+	f.DeletePod(framework.ClusterA, routeAgentPod.Name, framework.TestContext.SubmarinerNamespace)
+
+	NewRouteAgentPod := f.AwaitRouteAgentPod(framework.ClusterA)
```

Comment:
You should verify a new pod is started similar to `AwaitNewSubmarinerEnginePod`.

```suggestion
	newRouteAgentPod := f.AwaitNewRouteAgentPod(framework.ClusterA)
```